.gitignore

```diff
@@ -18,6 +18,7 @@ out
 # make sure the external config files are built 
 # on user's side each time, instead of using the ones from repo
 luthier/config/external
+luthier/src/test/resource
```

Comment:
We shouldn't ignore the resources directory. That may include
some configuration files that we *do* want to source control. Instead,
we should ignore just the JSON files in there. So something like:
`luthier/src/test/resources/`

Also, it should be `resources` not `resource`. Note the plural.

Reply:
Fixed. 
What I changed it to is that we ignore the current *Config.json files one by one, but not the entire dir.
This way we are a little bit more explicit when we think some config files should always be rebuilt, by adding that to the .gitignore list.

Reply:
It doesn't look like we're actually ignoring any of the *Config.json files, and they've all been added to version control
Is that intentional?